pages/governmentresumeguide.md

```diff
@@ -0,0 +1,75 @@
+#Government-style Resume Guide
+
+**Full legal name (First Middle Last)** <br>
+**City, State of Current Residence** <br>
+**Email address** <br>
+**Phone number** <br>
+
+**TECHNICAL SKILLS & TOOLS** <br>
+8+ years of experience: *List all the skills and tools you have experience with for more than 8
+years. If not applicable, do not include this section.* <br>
+4-7 years of experience: *List all the skills and tools you have experience with for 4-7 years. If
+not applicable, do not include this section.* <br>
+1-3 years of experience: *List all the skills and tools you have experience with for 1-3 years.* <br>
+
+**PROFESSIONAL EXPERIENCE** <br>
+*Start with your current employment or your most recent role and list all the professional
+experiences you’ve had in reverse chronological order. There needs to be a full chronology,
+meaning no breaks in time.* <br>
+
+*For your most relevant, recent and/or longest held positions you should list a minimum of 8-10
+bullet points. After that, it is ok to gradually lessen the number of bullet point descriptions.* <br>
+
+*Unrelated jobs held many years ago do not need more than 1 or 2 bullet points, however,
+transferrable skills such as management, communication, and problem solving should be
+highlighted, as those are relevant skills.* <br>
+
+*See below for employment history formatting.*
+<br>
+<br>
+**Role/title, Company name** <br>
+**City, State if within US or City, Country if outside of US** <br>
+**Duration of employment (Start MM/YYYY - End MM/YYYY or “Present”)** <br>
+**“Full-Time” or “Part-Time”, Hours per week: ##** <br>
```

Comment:
Great! Thank you. I don't think I can make the change here but will find out where I can take out ##